.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npm run lint:fix && git add .
+npm run lint:fix
```

Comment:
why remove git add?

Reply:
Removing `git add .` helps us to choose the specific files we want for commit and it stops us from adding files we don't want by mistake, automatically adding all changed files to the commit can be somehow  dangerous as you might unintentionally commit files that you didn't intend to